template/altium/COPYING

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2015, Linaro, Inc.
```

Comment:
This copyright should be 2017 and it should include yourself.
Linaro does not own the copyright as you created the template.